sdk/v1/client.go

```diff
@@ -129,7 +129,7 @@ func (cli *Client) PatchJob(ctx context.Context, headers client.Headers, jobID s
 
 	respHeader, _, err := cli.callReindexAPI(ctx, path, http.MethodPatch, headers, payload)
 	if err != nil {
-		return client.RespHeaders{}, err
+		return &client.RespHeaders{}, err
```

Comment:
error will return nil, i.e. no respHeaders

Reply:
I think, all error should return nil, not an empty ETag string / Json. The idea is client should not get any info/signature for ETag or respHeader in error scenario. Pls look at existing implementation in GetTask method.